src/Microsoft.DotNet.Build.Tasks.Feed/BlobFeed.cs

```diff
@@ -261,7 +274,20 @@ public bool IsSanityChecked(IEnumerable<string> items)
                     }
                 }
             }
-            string item = GenerateRootServiceIndex(relativePath);
+            string sasToken = string.Empty;
+            if (isSecure) {
+                var sasTokenCollection = Regex.Matches(SASToken, sasTokenRegex);
+                foreach (Match configGroup in sasTokenCollection)
+                {
+                    string svValue = configGroup.Groups[1].Value;
+                    string srValue = configGroup.Groups[2].Value;
+                    string spValue = configGroup.Groups[3].Value;
+                    string seValue = configGroup.Groups[4].Value;
+                    string sigValue = configGroup.Groups[5].Value;
+                    sasToken = $"/sv/{svValue}/sr/{srValue}/sig/{sigValue}/se/{seValue}/sp/{spValue}/";
```

Comment:
There are two caveats here:
- Because of the % issue, the sig value, which ends in %3D, has that bit chopped off when passed into the SaS URL.  In addition, the resulting sigValue can't have any %'s in it.  We might just want to assert that here.
- Also because of the % issue, the end date must have a 12:00am time.   The proxy appends T07%3A00%3A00Z automatically.  We should assert the input SaS has that value at the end of seParam.
- The SaS should be generated without a start time.

Reply:
I'll ensure that the regex doesn't include % and only check the date from the end time.

Reply:
Also the signature is Base64 encoded so we should only see [^A-Za-z0-9+/=] in the character set.